hack/e2e-suite/services.sh

```diff
@@ -34,6 +34,24 @@ source "${KUBE_VERSION_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
 
 prepare-e2e
 
+echo "Creating test namespace ${NAMESPACE}"
+${KUBECTL} create -f - << __EOF__
+{
+    "kind": "Namespace",
+    "apiVersion": "v1beta3",
+    "metadata": {
+        "name": "${NAMESPACE}"
+    },
+    "spec": {
+        "finalizers": [
+            "kubernetes"
```

Comment:
It is needed but it's applied on the server if omitted on create

Sent from my iPhone

> On Jun 2, 2015, at 8:05 PM, Brian Grant <notifications@github.com> wrote:
> 
> In hack/e2e-suite/services.sh:
> 
> > @@ -34,6 +34,24 @@ source "${KUBE_VERSION_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
> >  
> >  prepare-e2e
> >  
> > +echo "Creating test namespace ${NAMESPACE}"
> > +${KUBECTL} create -f - << __EOF__
> > +{
> > +    "kind": "Namespace",
> > +    "apiVersion": "v1beta3",
> > +    "metadata": {
> > +        "name": "${NAMESPACE}"
> > +    },
> > +    "spec": {
> > +        "finalizers": [
> > +            "kubernetes"
> I don't think this finalizer is needed for a namespace other than default.
> 
> —
> Reply to this email directly or view it on GitHub.
>